tests/headers/template_alias_namespace.hpp

```diff
@@ -0,0 +1,13 @@
+// bindgen-flags: --enable-cxx-namespaces -- -std=c++14
```

Comment:
It's the exact same test that `tests/headers/template_alias.hpp`, just that flag changes, so you can see the output right away :)